code/controllers/subsystem/statpanel.dm

```diff
@@ -168,6 +168,7 @@ SUBSYSTEM_DEF(statpanels)
 		list("World Time:", "[world.time]"),
 		list("Globals:", GLOB.stat_entry(), text_ref(GLOB)),
 		list("[config]:", config.stat_entry(), text_ref(config)),
+		list("Datasystem Manager:", "Edit", text_ref(SysMgr)),  // not real controller, but we want to access it easily
```

Comment:
```suggestion
		list("Datasystem Manager:", "Edit", text_ref(SysMgr)),
```

comment is irrelevant because this list is full of things that aren't controllers